cmd/operator/main.go

```diff
@@ -119,32 +110,40 @@ func main() {
 }
 
 func runCLI(logger *logrus.Logger, cfg *config.Config) {
```

Comment:
this probably can return an `error`. use `log.fatal`s sparingly because they ignore all call stack and finish the process immediately.

if you use `return`s as much as we can, we can do `log.fatal` in max 1-2 places, and have a chance to fail with either just the error message and potentially also with the callstack in debug mode.

Reply:
That's a good point. I'll change it once we start handling several services. The `log.Fatal` when rollout fails will go away since rollout of service A shouldn't probably affect rollout of service B. 

But yeah, there are more cases where we can error entirely (e.g. fail to filter targeted services)